config/locales/pl.yml

```diff
@@ -1,6 +1,17 @@
 pl:
   auth_admin_alert: 'Strona dostępna tylko dla administratorów'
 
+  mileages:
```

Comment:
We dont' need this translation.
```
4.5 Translations for Active Record Models
You can use the methods Model.model_name.human and Model.human_attribute_name(attribute) to transparently look up translations for your model and attribute names.
```
http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models